package.json

```diff
@@ -131,18 +132,16 @@
     "dist": "yarn dist:clean && yarn rollup:build && yarn proxy:build:release && electron-builder --publish never",
     "dist:clean": "rm -rf ./dist && mkdir ./dist && yarn proxy:clean && yarn rollup:clean",
     "dist:notarize": "NOTARIZE=true yarn dist",
-    "electron:start": "wait-on ./public/bundle.js && wait-on ./native/main.comp.js && wait-on tcp:17246 && NODE_ENV=development electron .",
+    "electron:start": "wait-on ./public/bundle.js ./native/main.comp.js && NODE_ENV=development electron .",
     "svelte:check": "svelte-check",
     "svelte:check:watch": "svelte-check --watch",
     "rollup:clean": "rm -rf public/bundle.* && rm -f native/main.comp.js",
     "rollup:build": "yarn rollup:clean && rollup -c",
     "rollup:watch": "yarn rollup:clean && rollup -c -w",
     "typescript:check": "tsc --noEmit && tsc --noEmit --project cypress && svelte-check",
-    "proxy:build": "cd proxy && cargo build --all-features --all-targets",
```

Comment:
Why are these flags being removed?

Reply:
They're not used anywhere.

Reply:
`--all-targets` seems fine to remove since we're building at the top of the workspace, so I believe it will build all the workspace targets.

`--all-features` I'm a little hesitant about. I suppose we don't have any features _now_ but if we did introduce feature flags later that would be different. No biggy for now though.

Reply:
Hm, I think I might have misunderstood what these flags do. Thanks for pointing it out, I added them back, I guess they do no harm and prevent us from tripping over in the future if we change something in proxy.

https://github.com/radicle-dev/radicle-upstream/pull/1489/commits/4cc40f5437174a0f7162a4e9e4d7641d70ddc738